python/pyiceberg/table/snapshots.py

```diff
@@ -96,6 +96,12 @@ class Snapshot(IcebergBaseModel):
     summary: Optional[Summary] = Field()
     schema_id: Optional[int] = Field(alias="schema-id", default=None)
 
+    def __str__(self) -> str:
```

Comment:
It would be great to have operation in here as well.

Reply:
Good one, added it! 👍🏻